mne/io/__init__.py

```diff
@@ -25,7 +25,7 @@
 from .bti import read_raw_bti
 from .edf import read_raw_edf
 from .egi import read_raw_egi
-from .kit import read_raw_kit
+from .kit import read_raw_kit, read_epochs_kit
```

Comment:
@agramfort always talks about having `mne.io` as Raw data reading -- perhaps `read_epochs_kit` should go right into the `mne` namespace along with `read_epochs`.